src/modules/dashboard/components/Dialogs/ControlSafeDialog/TransactionTypesSection/TransferFundsSection.tsx

```diff
@@ -87,23 +87,28 @@ const TransferFundsSection = ({
     setBalanceError('');
     if (safe) {
       setIsLoadingBalances(true);
+      let baseUrl: string;
       try {
         const chainName = getChainNameFromSafe(safe.profile.displayName);
```

Comment:
Move `getChainNameFromSafe` to own try catch in case the user inputs manually into the safe picker, which will throw an error we can catch and set the error to `invalidSafeError`